src/Effect.js

```diff
@@ -147,8 +147,15 @@ class Effect extends EventEmitter {
    * @return {number} attribute modified by effect
    */
   modifyAttribute(attrName, currentValue) {
-    const modifier = (this.modifiers.attributes[attrName] || (_ => _)).bind(this);
-    return modifier(currentValue);
+    let modifier = _ => _;
```

Comment:
Yeah, looks like it's similar to the previous setup, except that `modifiers.attributes` can be either a function, that probably contains a switch/case to determine how to modify the attributes, or it can be an object with keys mapped to functions that modify the attributes. Pretty nifty, actually. Is this all documented in the effects docs?